scripts/install_from_scratch.sh

```diff
@@ -3,14 +3,29 @@
 # Exit if any step in this install script fails.
 set -e
 
-echo "Beginning clean install of Ambuda."
+if [ -f data ] || [ -f env ] || [ -f node_modules ] || [ -f .env ] || [ -f database.db ]; then
+cat << "EOF"
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ERROR ERROR ERROR ERROR ERROR ERROR ERROR ERROR ERROR ERROR ERROR ERROR ERROR ERROR
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some of the files in your directory were created during a previous install. Our
+install script does not know how to treat these files.
+
+If you want to permanently delete these old files and install Ambuda from
+scratch, please run the following command:
 
+    make destructive-clean install
+
+EOF
+    exit 1
```

Comment:
There should be a way for someone to continue with the `make install` installation process if they don't want to run `make destructive-clean install` (otherwise, why would we create a separate script for `make destructive-clean install` in the first place?). Maybe just show a warning here (as opposed to quitting) and allow the user to continue with the installation, if they know what they're doing?

Reply:
sgtm -- this was a quick fix because I didn't feel like looking into parsing and sanitizing a user confirmation in bash. Is there a standard pattern you like here?

Reply:
I recently heard of https://github.com/charmbracelet/gum — haven't tried it yet but it seems cool :-)

Reply:
One option is to use libraries that we install in our Python or Node deps. So, we would bootstrap an initial setup with `pip install` in a shell script then run a script like `python install_from_scratch.py` that we could polish thoroughly.

Reply:
@akprasad How about this?

```bash
python3 -c "exit(0) if input('Are you sure you want to continue? (y/n): ') == 'y' else exit(1)"
```

Reply:
Works nicely! Doing a run through, will update the PR in a moment.